cmd/kube-apiserver/app/aggregator.go

```diff
@@ -95,28 +94,16 @@ func createAggregatorConfig(
 		return nil, err
 	}
 
-	var certBytes, keyBytes []byte
```

Comment:
should we make sure the files are readable at startup? a missing file error is easier to diagnose than weird anonymous request errors because the certs couldn't load in the background